README.md

```diff
@@ -11,4 +11,7 @@
 Follow these steps to build and run the application on your local device:
 
 - clone the repo to your local machine
-- run `make dev`
\ No newline at end of file
+- run `make dev`
+
+### Postman Collection
+- https://www.postman.com/robinhoodarmy/workspace/backend-api/overview
```

Comment:
Couldn't find workspace 👀 
> The workspace may have been deleted or the URL may be incorrect.

Reply:
made the collection public. can you try now. Also for write access, I have added the invite link in discord.